packages-exp/functions-exp/src/api.ts

```diff
@@ -55,6 +56,23 @@ export function getFunctions(
   return functionsInstance;
 }
 
+/**
+ * Modify this instance to communicate with the Cloudd Functions emulator.
+ *
+ * <p> Note: this must be called before this instance has been used to do any operations.
+ *
+ * @param host the emulator host (ex: localhost)
+ * @param port the emulator port (ex: 5001)
+ * @public
+ */
+export function useEmulator(
```

Comment:
Is it intended for the function to have the same name, `useEmulator`, in exp for Functions, Firestore, and Database?  Sorry if I missed it in the proposal, I don't think I saw anything on it. This would require the user to rename it on import if they're planning to use more than one, correct? e.g.
```
import { useEmulator as useFunctionsEmulator } from 'firebase/functions'
import { useEmulator as useDatabaseEmulator } from 'firebase/database'
```

Reply:
Ah yeah the proposal didn't cover `exp` since I didn't know about that at the time.  I see the issue.  I could either:

a) Omit `exp` for now
b) Go with `useFunctionsEmulator` as I think that's reasonable.

Reply:
And actually now that I think about there's no reason to carry forward a `@deprecated` method to the `exp` API at all, right?

Reply:
Oh right, makes sense, much simpler.